pageserver/src/metrics.rs

```diff
@@ -100,6 +100,32 @@ pub(crate) static VEC_READ_NUM_LAYERS_VISITED: Lazy<Histogram> = Lazy::new(|| {
     .expect("failed to define a metric")
 });
 
+pub(crate) static CONCURRENT_INITDBS: Lazy<UIntGauge> = Lazy::new(|| {
+    register_uint_gauge!(
+        "pageserver_concurrent_initdb",
+        "Number of initdb processes running"
+    )
+    .expect("failed to define a metric")
+});
```

Comment:
nit: gauges are less useful than a counterpair (we have an abstraction for counterpair)

Reply:
I think it's fine for this. I'm just looking for a general signal of whether that semaphore is at capacity.